Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -129,57 +133,49 @@ - (instancetype)initWithFrame:(CGRect)frame {
  Microsoft Extension
 */
 - (instancetype)initWithFrame:(CGRect)frame xamlElement:(RTObject*)xamlElement {
-    // TODO: We're passing nil to initWithFrame:xamlElement: because we have to *contain* a _subview for padding/layout.
-    // Note: Pass 'xamlElement' instead, once we move to a *single* backing Xaml element for UIActivityIndicatorView.
-    if (self = [super initWithFrame:frame xamlElement:nil]) {
-        [self _initUIActivityIndicatorView:xamlElement];
+    if (self = [super initWithFrame:frame xamlElement:xamlElement]) {
+        [self _initUIActivityIndicatorView];
     }
 
     return self;
 }
 
 /**
  @Status Interoperable
+ @Note Frame will not be resized smaller than the contained ProgressRing, as this would obscure it.
```

Comment:
If we switch to "White Large Style", shouldn't we adjust our frame accordingly?

---
In reply to: [113080757](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113080757) [](ancestors = 113080757,113079902,113078524)